tests/cmdline/commands/test_setup.py

```diff
@@ -206,11 +206,15 @@ def test_setup_profile_uuid(self):
 
         profile_name = 'profile-copy'
         user_email = 'some@email.com'
+        user_first_name = 'John'
```

Comment:
Did this test work previously because these options are not required? Or because these options were not properly validated?

Reply:
it worked properly because the type for these options is `NON_EMPTY_STRING` so any string goes and `()` is cast to a string and then accepted 😅